fe/fe-core/src/main/java/com/starrocks/sql/optimizer/statistics/CachedStatisticStorage.java

```diff
@@ -237,8 +236,7 @@ public ColumnStatistic getColumnStatistic(Table table, String column) {
             return getDefaultColumnStatisticList(columns);
         }
 
-        Table statisticsTable = StatisticUtils.getStatisticsTable();
-        if (statisticsTable == null) {
+        if (!StatisticUtils.checkStatisticTableStateNormal()) {
             return getDefaultColumnStatisticList(columns);
```

Comment:
if the FE cache is right, but some BE done，we will don't the column statistics.

Reply:
Yeah, will return Unknow if statistic table doesn't work. Maybe we can update the logical？